ui/selectors/nft.ts

```diff
@@ -53,3 +62,14 @@ export const getNftContractsByAddressOnCurrentChain = createSelector(
     return nftContractsByAddressByChain[currentChainId] ?? {};
   },
 );
+
+export const selectAllNftsFlat = createSelector(
+  getNftsByChainByAccount,
+  (nftsByChainByAccount) => {
+    const nftsByChainArray = Object.values(nftsByChainByAccount);
+    return nftsByChainArray.reduce((acc, nftsByChain) => {
+      const nftsArrays = Object.values(nftsByChain);
+      return acc.concat(...nftsArrays);
+    }, [] as Nft[]);
```

Comment:
Nit: The cast used here is unnecessary, we can instead pass `Nft[]` in as the type parameter:

```suggestion
    return nftsByChainArray.reduce<Nft[]>((acc, nftsByChain) => {
      const nftsArrays = Object.values(nftsByChain);
      return acc.concat(...nftsArrays);
    }, []);
```

Reply:
Done.